### DISCUSSION TITLE
【柱状图/折线图】当x轴是日期类型，某些label不显示

### DISCUSSION THREAD

**Discussion by May-yfm** [antvis/G2#5851] [Q&A]
### 问题描述

柱状图、条形图、折线图都遇到此问题，当x轴是日期类型，Axis的某些label缺失

### 重现步骤

1. g2 v4版本
2. https://g2-v4.antv.vision/zh/examples/case/column#column2
    以官网柱状图为例，修改数据为11位日期格式，Axis的某些Label不显示
3. 查看附件图片

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

### 屏幕截图或视频（可选）

![bug01](https://github.com/antvis/G2/assets/66411368/f09bb2b9-b6e4-4ca4-8e83-a2df33e80fb5)
![bug02](https://github.com/antvis/G2/assets/66411368/e82e9bb8-5b54-47c3-8264-635559d3707e)

**Comment by hustcc** [comment]
代码贴一下，不要截图。

**Comment by pearmini** [comment]
这个应该是 label layout 导致的，关闭一下 hideOverlap
